bin/ream/src/cli/mod.rs

```diff
@@ -33,6 +33,26 @@ pub struct NodeConfig {
     )]
     pub network: Arc<NetworkSpec>,
 
+    /// HTTP port number
+    #[arg(long, default_value_t = 5052)]
+    pub http_port: usize,
+
+    /// HTTP bind address
+    #[arg(long,default_value_t = String::from("127.0.0.1"))]
+    pub http_address: String,
+
+    /// Allow CORS
+    #[arg(long, action = ArgAction::SetTrue)]
+    pub http_allow_origin: bool,
+
+    /// discv5 listening port
+    #[arg(long, default_value_t = 8080)]
```

Comment:
could you make these default values constants and put the constants at the top of the file